pkg/finder/fsfinder.go

```diff
@@ -116,13 +117,6 @@ func (fsf FileSystemFinder) Find() ([]FileMetadata, error) {
 }
 
 // isExtensionExcluded returns true if extension exists in exclude list.
-// TODO: refactor to slices.Contains when project will be updated to go 1.21.
 func (fsf FileSystemFinder) isExtensionExcluded(ext string) bool {
```

Comment:
This entire function can be deleted now with the slices library supporting contains. The call to this function on line 94 can be replaced with `slices.Contains[[]string](fsf.ExcludeFileTypes, ext)`

Make sure to remove any unit tests that were written for `isExtensionExcluded`

Reply:
@kehoecj I did exactly this in the first commit. Reverted thinking we may need to keep the abstraction alive. Updated again. Thanks.